wpilibNewCommands/src/main/native/include/frc2/command/Commands.h

```diff
@@ -168,10 +175,15 @@ CommandPtr DeferredProxy(wpi::unique_function<Command*()> supplier);
  * called when the command is initialized.
  *
  * @param supplier the command supplier
+ * @deprecated The ProxyCommand supplier constructor has been deprecated in
+ * favor of directly proxying a {@link DeferredCommand}, see ProxyCommand
+ * documentaion for more details. To replace this, use
+ * `Defer(supplier).AsProxy()`.
```

Comment:
This note about using `Defer(supplier).AsProxy()` should be in the other overload too (`DeferredProxy(wpi::unique_function<Command*()>)`), and optionally make the wording in here and in Java exactly match up. (Currently Java says "As a replacement, consider using ..." while C++ says "To replace this, use ...")